limits/imap.go

```diff
@@ -54,10 +54,10 @@ func (i IMAP) CheckUIDValidity(uid imap.UID) error {
 
 func DefaultLimits() IMAP {
 	return IMAP{
-		maxMailboxCount:           math.MaxUint32,
-		maxMessageCountPerMailbox: math.MaxUint32,
-		maxUIDValidity:            math.MaxUint32,
-		maxUID:                    math.MaxUint32,
+		maxMailboxCount:           math.MaxInt32,
```

Comment:
The value still needs to be `MaxUint32 (4294967295)`. What I suggested originally is to replace the data type of these variables from`int` to `uint32`. This should work for 32bit systems.

Reply:
Ah, misread your suggestion; fixed.